src/ansys/dpf/core/server.py

```diff
@@ -388,3 +390,46 @@ def get_or_create_server(server: BaseServer) -> Union[BaseServer, None]:
     if server:
         return server
     return _global_server()
+
+
+def available_servers():
+    """Searches all available installed DPF servers in the current machine
+    Binds new functions to the server module which helps to choose the appropritae/suitable version
+
+    after this call, you can do the following:
```

Comment:
```suggestion
    """Searches all available installed DPF servers on the current machine.
    
    This method binds new functions to the server module, which helps to choose the appropriate version.

    After calling this method, you can do the following:
```
There is no list of things you can do? Is this just not completed yet?